ramls/item.json

```
@@ -39,6 +39,10 @@
       "type": "string",
       "description": "Unique inventory control number for physical resources, used largely for circulation purposes"
     },
+    "shelvingOrder": {
+      "type": "string",
+      "description": "Shelving order automatically generated form Call Number (R/O for manually editing)"
```

I suggest we use the description provided by Charlotte from [UIIN-816](https://issues.folio.org/browse/UIIN-816)

```suggestion
      "description": "A system generated normalization of the call number that allows for call number sorting in reports and search results"
```